polymer/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>abe-polymer2</title>
+    <meta name="description" content="test of Polymer">
+
+    <!-- See https://goo.gl/OOhYW5 -->
+    <link rel="manifest" href="/manifest.json">
+
+    <script src="/polymer/node_modules/webcomponents.js/webcomponents.min.js"></script>
```

Comment:
It's better to conditionally load the polyfills using https://github.com/PolymerElements/polymer-starter-kit/blob/e96167dac682c847008aee8ed40f2040b361da8f/index.html#L87-L88

Reply:
I am using node modules instead of bower and i can't find webcomponents-loader.js inside webcomponents.js